examples/browser/Dockerfile

```diff
@@ -24,20 +24,16 @@
 # 
 # 5. There should now be a 3000 link at the top of your PWD window. Click on it
 #    to try out the Theia browser example!
-#    
```

Comment:
I would be in favor of cleaning-up the documentation at the top of this file. Keeping a minimum on how to use it (e.g. mention building theia-dev image first, then how to build and run the theia image.

Reply:
I've cleaned up the doc part, please review

Reply:
Looks good, thanks.